### PR TITLE
[RF] AT_09.01_004 | <My Views>Create new view with Job Filters

### PR DESCRIPTION
**Pull request by ViktoriyaED**
RF - https://trello.com/c/Lk1ix3Bw/1024-rf-at0901004-my-viewscreate-new-view-with-job-filters-to-add-jobs-from-an-existing-view

### CODE REVIEW

File: cypress/e2e/myViewCreateNewBasedOnExisting.cy.js [hunk 1]
```diff
@@ -11,32 +11,33 @@ describe('myViewCreateNewBasedOnExisting', () => {
...
         cy.get('#ok-button').click()
         cy.get('button[name=Submit]').click();
-    }    
+    }
+    
+    function createFreestyleJobsInTHeFolder(jobName, folderName) {
+        cy.get(`[href="/job/${folderName}/newJob"]`).click();
+        cy.get('#name').type(jobName);
+        cy.get('span.label').contains('Freestyle project').click();
+        cy.get('#ok-button').should('have.text', 'OK').click();
```
Comment (Rom-T) [inline cypress/e2e/myViewCreateNewBasedOnExisting.cy.js:20]: Please move `OK` in the fixtures

File: cypress/e2e/myViewCreateNewBasedOnExisting.cy.js [hunk 2]
```diff
@@ -11,32 +11,33 @@ describe('myViewCreateNewBasedOnExisting', () => {
...
+        cy.get('button[name=Submit]').click();
+        cy.get('a[href*="/job/"]').contains(folderName).click();
+        cy.get('#main-panel h1').then(($el) => {
+            const header = $el.text().trim();
+            expect(header).to.equal(folderName)
+        })
```
Comment (Rom-T) [inline cypress/e2e/myViewCreateNewBasedOnExisting.cy.js:26]: I do not quite understand why we need to check the header. If it were not that folder, then the test would have failed at the last step when the folder's breadcrumb would have been clicked.

File: cypress/e2e/myViewCreateNewBasedOnExisting.cy.js [hunk 3]
```diff
@@ -11,32 +11,29 @@ describe('myViewCreateNewBasedOnExisting', () => {
         cy.get('span.label').contains('Folder').click();
```
Comment (Rom-T) [inline cypress/e2e/myViewCreateNewBasedOnExisting.cy.js:11]: Please move `Folder` in the fixtures

File: cypress/e2e/myViewCreateNewBasedOnExisting.cy.js [hunk 4]
```diff
@@ -11,32 +11,29 @@ describe('myViewCreateNewBasedOnExisting', () => {
...
         cy.get('#ok-button').click()
         cy.get('button[name=Submit]').click();
-    }    
+    }
+    
+    function createFreestyleJobsInTHeFolder(jobName, folderName) {
+        cy.get(`[href="/job/${folderName}/newJob"]`).click();
+        cy.get('#name').type(jobName);
+        cy.get('span.label').contains('Freestyle project').click();
```
Comment (Rom-T) [inline cypress/e2e/myViewCreateNewBasedOnExisting.cy.js:19]: Please move `Freestyle project` in the fixtures or to use a locator without resorting to a contains, then I would.
cy.get('li[class="hudson_model_FreeStyleProject"]').click();

File: cypress/e2e/myViewCreateNewBasedOnExisting.cy.js [hunk 5]
```diff
@@ -11,32 +11,29 @@ describe('myViewCreateNewBasedOnExisting', () => {
...
-        cy.get('.jenkins-breadcrumbs__list-item').find('[href="/"]').click();
-
+        createFolder(myView.FolderName1);   
+        createFreestyleJobsInTHeFolder(myView.JobName1, myView.FolderName1); 
+        createFreestyleJobsInTHeFolder(myView.JobName2, myView.FolderName1); 
+        createFreestyleJobsInTHeFolder(myView.JobName3, myView.FolderName1);        
+        cy.get('a[href="/"]').contains('Dashboard').click();
         createFolder(myView.FolderName2);
-        cy.get('.jenkins-breadcrumbs__list-item').find('[href="/"]').click();
+        cy.get('a[href="/"]').contains('Dashboard').click();
```
Comment (Rom-T) [inline cypress/e2e/myViewCreateNewBasedOnExisting.cy.js:32]: Please move `Dashboard` in the fixtures

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by InnaRomankova — APPROVED** [review]

**Pull request merged**